frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -75,18 +78,23 @@ export function ProjectList() {
     slug = "",
     page = 1,
     per_page = 10,
-    sort_by = ["-updated_at"]
+    sort_by = ["-updated_at"],
+    status = "Active"
   } = Route.useSearch();
   const [debouncedValue] = useDebounce(slug, 300);
 
-  const query = useProjectList({
-    slug: debouncedValue,
-    page,
-    per_page,
-    sort_by
-  });
   const navigate = useNavigate();
 
+  const query =
+    status === "Archived"
+      ? useArchivedProject()
+      : useProjectList({
+          slug: debouncedValue,
+          page,
+          per_page,
+          sort_by
+        });
```

Comment:
This will cause bugs. What you need to do is disable one or the other when the status is different, using this : https://tanstack.com/query/latest/docs/framework/react/guides/disabling-queries